packages/vuetify/src/components/VStepper/VStepper.sass

```diff
@@ -192,6 +192,13 @@
         margin: $stepper-alt-labels-header-divider
         align-self: flex-start
 
+    .v-stepper__label
+      max-width: 10vw
```

Comment:
Why this value?

Reply:
To avoid collision of labels. When using the value `10vw`, saved the correct tab size

Reply:
I don't know if this is the correct solution, but if it is the value should be probably extracted to the sass var

Reply:
@jacekkarczmarczyk 
I understand that this is not a 100% correct solution, but I did not come up with how to do it more correctly =)
I extract the value as a variable and leave the decision to reviewers